src/insider-threats/techniques-tactics-and-procedures.md

```diff
@@ -0,0 +1,109 @@
+2.  The bottom line is that there is no single "safe" remote recruitment channel. You should vet your potential employees independently and **look beyond their skillset, which can often be impressive.** We have observed DPRK IT Workers joining companies by utilizing all the channels mentioned above. Avoiding recruiting a DPRK IT Worker is solely dependent on the quality of your hiring process; do not rely only on outsourced solutions.
+
+#### Am I Interviewing a DPRK IT Worker?
+1.  The list below can serve as a guide for avoiding hiring a DPRK IT Worker and as an exit-interview guide for verifying if an employee is a DPRK IT Worker.
+2.  Before discussing the heuristics: **Don't over-focus on stereotypical DPRK IT worker characteristics, such as being an Asian male in their 20s or 30s, using specific headphones, or obscuring their background. What you are looking for is evidence of misrepresentation and fraud. Job fraud is not exclusive to North Koreans - approach it as such.**
+3.  Below is a list of 'red flags' that can help you evaluate with higher confidence if you are dealing with a DPRK IT Worker. However, always keep the above point in mind! For every 'obvious' DPRK IT Worker, there will be some who are able to evade most, if not all, of the following flags:
+    1.  **Is their video on?** If not, insist they enable their camera.
+        1.  Some DPRK IT Workers have no problem presenting as themselves (without a facilitator, AI, or disabled video). See if you recognize anyone from this [Collection of DPRK IT Workers photos](https://x.com/browsercookies/media).
+    2.  **Is the background visible or obscured?** An obscured background isn't always a red flag, but it can be helpful for the final assessment.
+        1.  DPRK IT Workers love "stock" backgrounds. You'll often find them using the "Golden Gate Bridge" as a backdrop. Other times, you may notice them sitting in low-quality housing where the workspace is only divided by temporary walls or rugs.
+    3.  Read: [Detect if it's an AI-generated face or deepfake](https://unit42.paloaltonetworks.com/north-korean-synthetic-identity-creation/).
+    4.  **Start with small talk** relevant to their claimed nationality and location. This is especially important if you're dealing with someone presenting as an Asian man but providing, for example, Latin American documents/names while claiming to live in Southeast Asia (an identity mismatch). **Note: While a smaller number of female DPRK IT workers have been identified, they do exist. In other cases, you might encounter hired female facilitators. If the candidate claims to be in your country, offer an on-site interview. Beware, DPRK IT workers may send a stand-in. Verify that the person attending is the same one from the call.**
```

Comment:
We use the term 'facilitator' a lot, I suggest updating our definition of 'mule' in the earlier docs to use the term 'facilitator' instead and can include 'mule' in there as a type of facilitator.